### ISSUE TITLE
window warning in rolling_min()

### ISSUE THREAD

**Issue by mickey3721** [pola-rs/polars#8446]
### Polars version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://pypi.org/project/polars/) of Polars.

### Issue description

addtional warning message show when running Series attribute .rolling_min()
[src\lazy\dsl.rs:1363] window_size = "xxx"

### Reproducible example

```python
print(pl.Series([1,2,3,4,5,6,7,8,9,10]).rolling_min(window_size=3))

return:

shape: (10,)
Series: '' [i64]
[
	null
	null
	1
	2
	3
	4
	5
	6
	7
	8
]
[src\lazy\dsl.rs:1363] window_size = "3i"
```

### Expected behavior

check and fix

### Installed versions

<details>

```
---Version info---
Polars: 0.17.7
Index type: UInt32
Platform: Windows-10-10.0.23440-SP0
Python: 3.11.3 (tags/v3.11.3:f3909b8, Apr  4 2023, 23:49:59) [MSC v.1934 64 bit (AMD64)]
---Optional dependencies---
numpy: 1.24.3
pandas: 2.0.0
pyarrow: 11.0.0
connectorx: <not installed>
deltalake: <not installed>
fsspec: <not installed>
matplotlib: 3.7.1
xlsx2csv: <not installed>
xlsxwriter: <not installed>
```

</details>

**Comment by cmdlineluser** [comment]
A stray `dbg!()` call was left in:
https://github.com/pola-rs/polars/blob/main/py-polars/src/lazy/dsl.rs#L1363

**Comment by ritchie46** [comment]
Thanks fixed in #8450
